### ISSUE TITLE
Ignored custom eslint plugin rules

### ISSUE THREAD

**Issue by vladi-strilets** [idahogurl/vs-code-prettier-eslint#1]
Hello, I noticed a format bug with a custom plugin. 

Here I leave an example https://github.com/vladi-strilets/vs-code-prettier-eslint

Just clone the master, do yarn, go to the `src/test.js` and format with the **prettier-eslint extension**.

I have added a new plugin https://github.com/kusotenpa/eslint-plugin-destructuring-newline

The problem is that this code has no been formatted. 
```
const { a, b } = foo;
```

Expected result: 
```
const {
  a,
  b,
} = foo;
```
-----
I have installed prettier-eslint-cli to check if I was a problem with prettier-eslint, but it works well, you can check it by;
```
yarn prettier-eslint --write src/test.js
```
---- 
Any ideas, thanks!

**Comment by idahogurl** [comment]
@vladi-strilets Thank you for making an issue!! Thanks for making a reproducible example repo as well. I'll check into ASAP

**Comment by idahogurl** [comment]
@vladi-strilets What version of the `prettier-eslint-cli` did you install? `eslint-plugin-destructuring-newline@0.0.2` requires ESLint 7 and my plugin only supports `6.8.0` for the time being.

**Comment by vladi-strilets** [comment]
@idahogurl 
I found a stranger behavior with prettier-eslint.

When I pass the **text** param to the  `format` function, it returns a formated string but without applying the object restructuring rule. 
Meanwhile, passing the **filePath** prop of the file, the object restructuring rule is applied well. 

Here I have updated the example:
https://github.com/vladi-strilets/vs-code-prettier-eslint/blob/master/src/test.js 
Just `node src/test.js`

Anyway, I don't know why, but now my VSCode applies the rule for the **test2.js** file. However, in my other project, the same rule just underlines the problem but can't fix it. Maybe it's some config conflict. 

One more thing, is it possible to update eslint and prettier versions of the package? 

Thanks!
